cobertura/README.md

```diff
@@ -0,0 +1,48 @@
+# OPA Test Coverage to Cobertura Format
+
+Example code to transform the OPA test coverage JSON report to
+the [Cobertura](https://github.com/cobertura/cobertura/blob/master/cobertura/src/test/resources/dtds/coverage-04.dtd) coverage report format.
+
+## Why?
+
+Cobertura is one of the supported report formats for
+[Jenkins Coverage API plugin](https://github.com/jenkinsci/code-coverage-api-plugin), which is widely used.
+
+## Example
+
+Simply pipe the output of `opa test --coverage` into `opa eval` with the `simplecov.rego` file loaded:
```

Comment:
This doesn't seem like it's the right procedure for this integration.

Reply:
deleted.